website/src/views/JoinClass/JoinClass.jsx

```diff
@@ -107,7 +100,7 @@ class JoinClass extends Component {
     this.setState({ loading: true });
     let API_URL = '';
     let reqData = {};
-    if (selectedClass.isApproved === true) {
+    if (selectedClass.approved === true) {
```

Comment:
`if(selectedClass.approved)` Come on @hetobias  this is csc 103 logic

Reply:
@ayushyamitabh Take care of him

Reply:
@khelif96 your point?

Reply:
backend checks either way...